### DISCUSSION TITLE
How do I configure for a solid black background "#000000"

### DISCUSSION THREAD

**Discussion by cool-clay** [catppuccin/nvim#695] [Q&A] [answered]
I find these sort of black-gray, black-blue backgrounds not only hard on the eyes but they lack the punch to make the colors pop out, especially with pastels.  That and it is hard to read especially because I keep my screen brightness low to save my eyes in that I spend allot of time at the screen.
I use Lazy neovim with lua settings.

**Comment by rewhile** [comment, marked as answer]
```lua
require("catppuccin").setup {
  color_overrides = {
    mocha = {
      base = "#000000",
      mantle = "#000000",
      crust = "#000000",
    },
  },
}
vim.cmd.colorscheme("catppuccin-mocha")
```

**Reply by cool-clay** [reply]
Thank you, works great.
I added it into my plugin folder plugins.capuccin like so.
-- catpuccin colorscheme
return {
  { "catppuccin/nvim", priority = 1000 },
  require("catppuccin").setup({
    color_overrides = {
      mocha = {
        base = "#000000",
        mantle = "#000000",
        crust = "#000000",
      },
    },
  }),
}
-- This, I put into my "config.autocmds" folder
vim.cmd.colorscheme("catppuccin-mocha")

I haven't set it as the default yet.. I think that lazyvim needs a few more lines of code at the start.
  {
    "LazyVim/LazyVim",
    opts = {
      colorscheme = "gruvbox",
    },

placed right after the return {

for others who have a similar setup.

Thanks
